jack-core/src/com/rapleaf/jack/transaction/DbPoolStatus.java

```diff
@@ -0,0 +1,10 @@
+package com.rapleaf.jack.transaction;
+
+public interface DbPoolStatus {
+
+  int getNumActive();
+
+  int getNumIdle();
+
+  int getNumWaiters();
+}
```

Comment:
Nitpick: newline in this interface is not consistent. Either remove the extra newline before `int getNumActive()`, or add a newline after `int getNumWaiters()`.